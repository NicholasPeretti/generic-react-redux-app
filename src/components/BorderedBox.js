import styled from 'styled-components'
import Style from '../style'

export default styled.div`
  border: 2px solid ${Style.colors.gray};
  padding: ${Style.spacing.padding3};
`
