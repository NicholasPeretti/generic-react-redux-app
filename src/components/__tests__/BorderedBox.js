import React from 'react'
import { shallow } from 'enzyme'
import PresentationalTest from '../../utils/tests/presentational'
import BorderedBox from '../BorderedBox'

describe('BorderedBox', () => {
  PresentationalTest(BorderedBox)

  test('Should match snapshot', () => {
    const wrapper = shallow(<BorderedBox/>)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
