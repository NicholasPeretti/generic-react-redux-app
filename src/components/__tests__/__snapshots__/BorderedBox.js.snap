// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BorderedBox Should match snapshot 1`] = `"<div class=\\"sc-bdVaJa fLeUxE\\"></div>"`;
